omnigibson/prims/geom_prim.py

```diff
@@ -50,8 +50,9 @@ def _post_load(self):
         # run super first
         super()._post_load()
 
-        # By default, GeomPrim shows up in the rendering.
-        self.purpose = "default"
```

Comment:
Is this deletion intended?

```
        self.purpose = "default"
```

Reply:
Yes, it's intended. If we don't do this, all visual meta links (e.g. inside volumes) are forced to visibility.